packages/main/src/Tokenizer.ts

```diff
@@ -108,32 +112,50 @@ enum ClipboardDataOperation {
 	],
 })
 
-@event("token-delete", {
+/**
+ * Fired when a token is deleted (delete icon, delete or bacspace is pressed)
+ * @param {HTMLElement} ref DOM ref of the token to be deleted.
+ * @public
+ */
+@event<TokenizerTokenDeleteEventDetail>("token-delete", {
 	detail: {
 		ref: { type: HTMLElement },
 	},
 })
 
-@event("show-more-items-press", {
-	detail: {
-		ref: { type: HTMLElement },
-	},
-})
+/**
+ * Fired when nMore link is pressed.
+ * @public
```

Comment:
* Do we need this to be public for the first version?

Reply:
Done.